### PR TITLE
Release artifact-pull 1.0.0

### PR DESCRIPTION
**Pull request by hisaac**
TagCheck

https://github.com/bitrise-steplib/bitrise-step-artifact-pull/releases/1.0.0

### Description

This pull request adds the `artifact-pull` Step to the Bitrise steplib.

You can see the Step's source code here: https://github.com/bitrise-steplib/bitrise-step-artifact-pull

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/artifact-pull/step-info.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+maintainer: bitrise
```
Comment (russellbstephens) [inline steps/artifact-pull/step-info.yml:1]: what does this do?

Comment (hisaac) [reply steps/artifact-pull/step-info.yml:1]: Effectively, I don't know what it does, but conceptually it just denotes that this is a Step maintained by Bitrise, not by the community.

**Comment by russellbstephens** [comment]
Looks like we exercise the step yml with this custom script remotely

Screen Shot 2022-07-08 at 9 11 57 AM

**Review by russellbstephens — APPROVED** [review]

File: steps/artifact-pull/step-info.yml (hunk 1 above)
Comment (ofalvai) [reply steps/artifact-pull/step-info.yml:1]: Workflow editor displays a badge that it's officially maintained by Bittise.

Comment (hisaac) [reply steps/artifact-pull/step-info.yml:1]: Ah gotcha. Thanks for the info 🙇‍♂️

**Pull request merged**